python/lsst/sdm/tools/build_datalink_metadata.py

```diff
@@ -0,0 +1,131 @@
+"""From the Felis source files, build YAML metadata used by DataLink.
```

Comment:
Is the idea that no user should ever import lsst.sdm.tools but these are always command-line tools?

At the moment all the code in here is public and I don't know if that's your intent. If they are public functions it would likely be better to move the `main` to a scripts directory or put the trivial implementation directly into cli.py.

Reply:
My intention is that this package provides command-line tools, and the modules would not be imported by users.